compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/dma_loop_subsumption.mlir

```diff
@@ -561,9 +556,6 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
         scf.for %arg5 = %c0 to %c64 step %c1 {
           amdaie.npu.dma_cpy_nd %0([0, 0, 0] [2, 8, 16] [128, 16, 1], [] [] [])
         }
-        scf.for %arg6 = %c0 to %c1024 step %c1 {
```

Comment:
Instead of removing, could you update the checks to ensure that loop subsumption doesn't happen? And I think it's useful as well to add a new test with a loop with extent `max_repeat_count - 1`.

Reply:
Done.